acceptance/acceptance_test.go

```diff
@@ -217,9 +243,12 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 		coverDir = filepath.Join(coverDir, strings.ReplaceAll(dir, string(os.PathSeparator), "--"))
 		err := os.MkdirAll(coverDir, os.ModePerm)
 		require.NoError(t, err)
-		cmd.Env = append(os.Environ(), "GOCOVERDIR="+coverDir)
+		cmdEnv = setEnv(cmdEnv, "GOCOVERDIR", coverDir)
```

Comment:
Could keep append() in place? It's more readable - everyone knows append(), no one knows what setEnv() does.